libtalloc/plan.sh

```diff
@@ -10,20 +10,26 @@ pkg_dirname="talloc-${pkg_version}"
 pkg_shasum="987c0cf6815e948d20caaca04eba9b823e67773f361ffafe676e24b953cc604b"
 pkg_lib_dirs=(lib)
 pkg_include_dirs=(include)
+pkg_pconfig_dirs=(core/pkg-config)
```

Comment:
The directory here should be `lib/pkgconfig` actually. `core/pkg-config`  is a `pkg_build_deps` dependency for the package itself :)